src/test/java/com/cloudbees/plugins/credentials/impl/DummyCredentials.java

```diff
@@ -35,15 +34,19 @@
 /**
  * A test credentials impl.
  */
-public class DummyCredentials extends BaseCredentials implements UsernamePasswordCredentials {
+public class DummyCredentials extends BaseStandardCredentials implements UsernamePasswordCredentials {
 
     private final String username;
 
     private final Secret password;
 
     @DataBoundConstructor
```

Comment:
Should the old constructor remain the `DataBoundConstructor` or should that move to the new one? Do we have UI tests that even use this? Should a `null` ID be a problem?

Reply:
there's a [jelly file](https://github.com/jenkinsci/credentials-plugin/blob/master/src/test/resources/com/cloudbees/plugins/credentials/impl/DummyCredentials/credentials.jelly) so anything doing a `configRundTrip` would be broken IIUC.

`<st:include page="id-and-description" class="${descriptor.clazz}"/>`

Reply:
Just found out that https://github.com/jenkinsci/credentials-plugin/blob/master/src/test/java/com/cloudbees/plugins/credentials/impl/DummyIdCredentials.java exists. I'll revert the change here and only adapt affected tests to use `DummyIdCredentials.java`